### DISCUSSION TITLE
Plex not recognizing agents

### DISCUSSION THREAD

**Discussion by Jay-Mega** [CodyBerenson/PGMA-Modernized#285] [General]
I am new to this, and I am following the readme...I put the PGMA folder in my plug-ins folder and restarted the PMS several times but none of the agents are showing up. What could I be doing wrong?

**Comment by CodyBerenson** [comment]
did you put the PGMA bundles and the _PGMA folder at the ROOT of your plug-ins folder?  Here's what my plug-ins folder looks like

![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/40373801/10c98e88-0147-4ed2-b26a-a682018956de)

**Reply by Jay-Mega** [reply]
I did! Looks like it was in still in the PGMA-Modernized-master my plug-ins folder which confused Plex. Thank you!
